finalPrep/modZeroHero.js

```diff
@@ -55,30 +96,67 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
+var scaryMonster = {
+  name: "Boo Boo",
+  smell: "rotting lettuce",
+  weight: 45,
+  citiesDestroyed: ["Portland", "Seattle", "Denver"],
+  luckyNumbers: [22, 33, 44, 55],
+  address: {
+    number: 123,
+    street: "Elm St.",
+    state: "Rhode Island",
+    zip: 34009
+  }
+}
+
 
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+
+
+var superHero = {
```

Comment:
This is an object literal (a very nice one, and you did a good job setting it up with the directions.) Double check the directions on line 115 for what was asked!